tests/phpunit/tests/http/wpGetHttpHeaders.php

```diff
@@ -2,6 +2,8 @@
 
 /**
  * @group http
+ * @group external-http
```

Comment:
In this file, only a couple of tests make external requests, others are mocked.

Instead of adding the group at the class level, add it to the individual test:

- `test_wp_get_http_headers_invalid_url` will need to include the `external-http` group
- `test_wp_get_http_headers_deprecated_argument` can change the URL from `does_not_matter` to `http://example.com`